spec/system/tools_language_filter_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+describe 'filter[lang] for Tools Pages' do # rubocop:disable RSpec/DescribeClass
```

Comment:
Why does this system spec's `describe` need a special Rubocop comment, but these existing system specs don't?

https://github.com/crimethinc/website/tree/main/spec/system

Reply:
@veganstraightedge https://github.com/crimethinc/website/blob/3407d59e9acddd2610e22a1d58fbb3b9990028af/.rubocop_todo.yml#L22-L30

Reply:
Ooooh. I see.

Reply:
i can put it there if it looks nicer. i honestly do not like that cop and would rather disable it

Reply:
How about we split the difference?

What if we move that from TODO to `.rubocop.yml`, but rather than disabling it whole cloth, we just `Exclude:` that system spec folder?

Reply:
sounds good to me